src/etl/notify/tests/test_notify_lambda.py

```diff
@@ -34,16 +44,18 @@
 ]
 
 
-def test_notify_lambda_with_state_machine_input_without_error_message():
-    from etl.notify import notify
-
-    assert notify.handler(event=STATE_MACHINE_INPUT_WITHOUT_ERROR_MESSAGE) == "pass"
+@pytest.mark.parametrize(
+    ["input", "output"],
+    [
+        (STATE_MACHINE_INPUT_WITHOUT_ERROR_MESSAGE, "pass"),
+        (STATE_MACHINE_INPUT_WITH_ERROR_MESSAGE, "fail"),
+    ],
+)
+def test_notify_lambda_with_state_machine_input_without_error_message(input, output):
```

Comment:
Name of this test needs changing as it is now testing both scenarios - with & without error message